### DISCUSSION TITLE
Multiplayer Issues

### DISCUSSION THREAD

**Discussion by Bobsshell** [TEHE-Studios/ExpandedHelicopterEvents#206] [Q&A]
I've had some issues going on in a multiplayer playthrough I have been doing with my friend. It seems that whenever an event occurs the sounds from it will loop constantly. One example would be hearing jets flying in the very distance for days on end, and the air raid siren sounding off for days on end as well. One time a helicopter flew in and then flew away to where I would assume it would despawn, but instead it would just fly in this one spot for days on end. Wondering if anyone has run into this issue or if it could be from my end, thanks.

**Comment by Chuckleberry-Finn** [comment]
#85
